tests/cpp/analysis/bls_analyzer_test.cpp

```diff
@@ -8,12 +8,25 @@
 #include "taichi/ir/snode.h"
 #include "taichi/ir/transforms.h"
 #include "taichi/struct/struct.h"
-#include "tests/cpp/struct/fake_struct_compiler.h"
 
 namespace taichi {
 namespace lang {
 namespace {
 
+class TestStructCompiler : public StructCompiler {
```

Comment:
Remove?

Reply:
Oops. not sure how that happened. reverted now